src/pages/home-page/home-page.scss

```
@@ -1,7 +1,55 @@
 @import "/src/styles/index";
 
 .home-page {
-    a {
-        @include flex-center;
+    padding: 40px 100px;
+    min-height: 100vh;
+    background-color: $light-text;
+    background-image: url("../../../static/images/background.png");
+    background-repeat: no-repeat;
+    &__info {
+        display: grid;
+        column-gap: 50px;
+        grid-template-columns: repeat(3, 1fr);
+        &__header h1{
+            @include font-style($font-size: 150px,$letter-spacing: -0.06em,$font-family: UbuntuCondensed);
```

```suggestion
            @include font-style($font-size: 150px, $letter-spacing: -0.06em, $font-family: UbuntuCondensed);
```
Кстати, мы не будем чётко определять единицы, в которых будем работать? А то я вижу прям чуть ли не все)
Ну и отступы бы)

---

все единицы брала с фигмы, поэтму так) а с отступами что?

---

пробелы в смысле, то есть, то нет) Мб в stylelint прописать просто и залинтфиксить

---

а окей, поправлю